crates/backend/src/ast.rs

```diff
@@ -39,7 +39,7 @@ pub struct Import {
 
 #[cfg_attr(feature = "extra-traits", derive(Debug, PartialEq, Eq))]
 pub enum ImportKind {
-    Function(ImportFunction),
+    Function(Box<ImportFunction>),
```

Comment:
We aren't really going for speed here, could this be left as is to keep it consistent ?

Reply:
:+1: